prow/cmd/sinker/main.go

```diff
@@ -124,15 +135,44 @@ func main() {
 		config:        cfg,
 	}
 
-	// Clean now and regularly from now on.
-	for {
-		start := time.Now()
-		c.clean()
-		logrus.Infof("Sync time: %v", time.Since(start))
-		if o.runOnce {
-			break
+	var wg sync.WaitGroup
+	wg.Add(1)
+
+	go func() {
+		defer wg.Done()
+		// Clean now and regularly from now on.
+		for {
+			start := time.Now()
+			c.clean()
+			logrus.Infof("Sync time: %v", time.Since(start))
+			if o.runOnce {
+				break
+			}
+			time.Sleep(cfg().Sinker.ResyncPeriod)
 		}
-		time.Sleep(cfg().Sinker.ResyncPeriod)
+	}()
+
+	if !o.runOnce {
+		// Return 200 on / for health checks.
+		http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {})
```

Comment:
We don't need this bit here

Reply:
I thought we had health check for all http severs.
Done.